lib/beaker/hypervisor/openstack.rb

```diff
@@ -152,27 +154,15 @@ def cleanup
       end
     end
 
-    # Enables root for instances with custom username like ubuntu-amis
-    #
-    # @return [void]
-    # @api private
-    def enable_root_on_hosts
-      @hosts.each do |host|
-        enable_root(host)
-      end
-    end
-
     # Enables root access for a host when username is not root
     #
     # @return [void]
     # @api private
     def enable_root(host)
-      if host['user'] != 'root'
-        copy_ssh_to_root(host, @options)
-        enable_root_login(host, @options)
-        host['user'] = 'root'
-        host.close
-      end
+      copy_ssh_to_root(host, @options)
+      enable_root_login(host, @options)
+      host['user'] = 'root'
```

Comment:
Any chance that this is going to execute on windows?  Then you'd want 'Administrator'?